app/workers/rapid_ready_for_decision/asthma_processor.rb

```diff
@@ -25,10 +25,19 @@ def query_and_assess_lighthouse
       { medications: medications }
     end
 
+    ASTHMA_KEYWORDS = RapidReadyForDecision::Constants::DISABILITIES[:asthma][:keywords]
```

Comment:
@yoomlam oh, so I guess this isn't an issue when it's defined on the class level (I guess because it can't be statically evaluated)